bin/improver-nbhood

```diff
@@ -35,15 +35,18 @@ import argparse
 
 import iris
 
-from improver.nbhood import BasicNeighbourhoodProcessing
+from improver.nbhood import NeighbourhoodProcessing
 
 
 def main():
     """Load in arguments and get going."""
     parser = argparse.ArgumentParser(
-    description='Apply basic weighted circle smoothing via ' +
-                'the BasicNeighbourhoodProcessing plugin ' +
+    description='Apply the requested kernel via ' +
+                'the NeighbourhoodProcessing plugin ' +
                 'to a file with one cube.')
+    parser.add_argument('kernel_method', metavar='KERNEL_METHOD',
+                        help='The kernel method to apply in neighbourhood'
+                             ' processing. Options: "circular".')
     group = parser.add_mutually_exclusive_group()
```

Comment:
Fair enough but how are we going to insure this list of options is updated each time a new kernel is added. It might be worth adding a comment to the Kernel class in nbhood.py to ensure that this doesn't get forgotten.